misc/init.d/cfengine3.in

```diff
@@ -451,6 +451,22 @@ cf_status_daemon()
     fi
 }
 
+cf_status_postgres_daemon() {
```

Comment:
If `cf_status_postgres_daemon` is only used in https://github.com/cfengine/nova/pull/981, why don't you add it there instead? Makes testing easier as well.